Generator/Assets/SeedData.cs

```diff
@@ -16,7 +16,7 @@ public class SeedData
         // See <add_documentation_reference_here> for the flowchart for
         // determining if you should increment the major or minor version.
         public static readonly UInt16 VersionMajor = 1;
```

Comment:
I think we should update to version 2 of seed

Reply:
I did that already. I updated the minor version to 2 so now the seed will be version 1.2. We are still maintaining backwards compatibility, so a version 2.0 is not needed at this time.